<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>991</width>
      <height>741</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Задание 1</string>
     </attribute>
     <widget class="QListWidget" name="list_answer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>321</width>
        <height>192</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Юпитер</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Марс</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Земля</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Нептун</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Венера</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="lb_answer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Третья планета от солнца?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_answer">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>70</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ответить</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_correct">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>170</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Задание 2</string>
     </attribute>
     <widget class="QTableWidget" name="table_sum">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>521</width>
        <height>191</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>5</number>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QPushButton" name="bt_sum">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Сумма</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_sum">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сумма = </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Задание 3</string>
     </attribute>
     <widget class="QComboBox" name="cb_num">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Квадраты чисел</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Кубы чисел</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Числа</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="bt_num">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Заполнить</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tb_num">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>331</width>
        <height>121</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>3</number>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Задание 4</string>
     </attribute>
     <widget class="QTableWidget" name="tb_sum_one">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>351</width>
        <height>291</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <row/>
      <column/>
     </widget>
     <widget class="QPushButton" name="bt_addrow_tb1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить строку</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_addcol_tb1">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить столбец</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_delrow_tb1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить строку</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_delcol_tb1">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>330</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить столбец</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_delrow_tb2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>330</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить строку</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_addrow_tb2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить строку</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_addcol_tb2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить столбец</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_delcol_tb1_2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>330</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить столбец</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tb_sum_two">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>351</width>
        <height>291</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <row/>
      <column/>
     </widget>
     <widget class="QTableWidget" name="tb_all">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>380</y>
        <width>591</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_random_tb1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Заполнить первую матрицу</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_random_tb2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Заполнить вторую матрицу</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_sum_matrix">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>500</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Суммировать матрицы</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
